src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -4135,5 +4135,32 @@ public class Methods2
             var expectedDescription = $"void Methods1.Do(string x)";
             VerifyWithReferenceWorker(markup, MainDescription(expectedDescription));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        [WorkItem(4868, "https://github.com/dotnet/roslyn/issues/4868")]
+        public void QuickInfoExceptions()
+        {
+            Test(@"
+using System;
+namespace MyNs
+{
+    class MyException1 : Exception { }
+    class MyException2 : Exception { }
+    class TestClass
+    {
+        /// <exception cref=""MyException1""></exception>
+        /// <exception cref=""T:MyNs.MyException2""></exception>
+        /// <exception cref=""System.Int32""></exception>
+        /// <exception cref=""double""></exception>
+        /// <exception cref=""Not_A_Class_And_Therefore_Not_Listed""></exception>
+        void M()
+        {
+            M$$();
+        }
+    }
+}
+",
+                Exceptions($"\r\n{WorkspacesResources.Exceptions}\r\n  MyException1\r\n  MyException2\r\n  double\r\n  int"));
+        }
```

Comment:
VS 2013 simply displays the types as listed in the doc comment, e.g.,

    MyException1
    MyNs.MyException2,
    System.Int32,
    double
    Not_A_Class_And_Therefore_Not_Listed

Having thought about the difficulties in dealing with sorting the fully-qualified names vs. sorting the simplified names (which could change with each new `using`), I'm leaning towards no sorting at all and simply using the declaration order, but still letting the symbol display service simplify names where possible, e.g.,

    MyException1
    MyException2
    int
    double

But there's still the issue of what to do with exceptions that didn't parse as a type.  The doc comment author thought it important to include an exception type here, but why?